File: Data Structures & Algorithms/climbing-stairs/submission-0.java
class Solution {
    public int climbStairs(int n) {
        int res = 0;

        if (n == 1 || n ==0) {
            return res + 1;
        }

        res = climbStairs(n-1) + climbStairs(n-2);
        return res;
        
    }
}

File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        res = []


        mp= {}
        for i, s in enumerate(strs):
            freq_arr = [0] * 26

            for c in s:
                freq_arr[ord(c) - ord('a')] += 1
            
            tp_freq = tuple(freq_arr)
            if tp_freq not in mp:
                mp[tp_freq] = [s]
            else:
                mp[tp_freq].append(s)
        
        
        return list(mp.values())